<?xml version="1.0" encoding="UTF-8" standalone="no"?>
<!DOCTYPE log SYSTEM "logger.dtd">
<log>
<record>
  <date>2024-02-06T07:48:10.121645500Z</date>
  <millis>1707205690121</millis>
  <nanos>645500</nanos>
  <sequence>0</sequence>
  <level>START</level>
  <class>com.kms.katalon.core.logging.XmlKeywordLogger</class>
  <method>startSuite</method>
  <thread>1</thread>
  <message>Start Test Suite : Test Suites/account_creation_errors/account_creation_errors</message>
  <nestedLevel>0</nestedLevel>
  <escapedJava>false</escapedJava>
  <property name="rerunTestFailImmediately">true</property>
  <property name="retryCount">3</property>
  <property name="name">account_creation_errors</property>
  <property name="description"></property>
  <property name="id">Test Suites/account_creation_errors/account_creation_errors</property>
</record>
<record>
  <date>2024-02-06T07:48:10.159350500Z</date>
  <millis>1707205690159</millis>
  <nanos>350500</nanos>
  <sequence>1</sequence>
  <level>RUN_DATA</level>
  <class>com.kms.katalon.core.logging.XmlKeywordLogger</class>
  <method>logMessage</method>
  <thread>1</thread>
  <message>hostName = Devil - LAPTOP-V2P5S4MK.mshome.net</message>
  <nestedLevel>0</nestedLevel>
  <escapedJava>false</escapedJava>
  <property name="hostName">Devil - LAPTOP-V2P5S4MK.mshome.net</property>
</record>
<record>
  <date>2024-02-06T07:48:10.161378100Z</date>
  <millis>1707205690161</millis>
  <nanos>378100</nanos>
  <sequence>2</sequence>
  <level>RUN_DATA</level>
  <class>com.kms.katalon.core.logging.XmlKeywordLogger</class>
  <method>logMessage</method>
  <thread>1</thread>
  <message>os = Windows 10 64bit</message>
  <nestedLevel>0</nestedLevel>
  <escapedJava>false</escapedJava>
  <property name="os">Windows 10 64bit</property>
</record>
<record>
  <date>2024-02-06T07:48:10.161587600Z</date>
  <millis>1707205690161</millis>
  <nanos>587600</nanos>
  <sequence>3</sequence>
  <level>RUN_DATA</level>
  <class>com.kms.katalon.core.logging.XmlKeywordLogger</class>
  <method>logMessage</method>
  <thread>1</thread>
  <message>hostAddress = 192.168.137.1</message>
  <nestedLevel>0</nestedLevel>
  <escapedJava>false</escapedJava>
  <property name="hostAddress">192.168.137.1</property>
</record>
<record>
  <date>2024-02-06T07:48:10.162583Z</date>
  <millis>1707205690162</millis>
  <nanos>583000</nanos>
  <sequence>4</sequence>
  <level>RUN_DATA</level>
  <class>com.kms.katalon.core.logging.XmlKeywordLogger</class>
  <method>logMessage</method>
  <thread>1</thread>
  <message>katalonVersion = 9.2.0.212</message>
  <nestedLevel>0</nestedLevel>
  <escapedJava>false</escapedJava>
  <property name="katalonVersion">9.2.0.212</property>
</record>
<record>
  <date>2024-02-06T07:48:10.463518500Z</date>
  <millis>1707205690463</millis>
  <nanos>518500</nanos>
  <sequence>5</sequence>
  <level>START</level>
  <class>com.kms.katalon.core.logging.XmlKeywordLogger</class>
  <method>startTest</method>
  <thread>1</thread>
  <message>Start Test Case : Test Cases/account_creation/invalid_name</message>
  <nestedLevel>1</nestedLevel>
  <escapedJava>false</escapedJava>
  <property name="remainingRetryCount">3</property>
  <property name="name">Test Cases/account_creation/invalid_name</property>
  <property name="description"></property>
  <property name="iteration"></property>
  <property name="id">Test Cases/account_creation/invalid_name</property>
  <property name="source">C:\Users\Devil\git\SI-GuidedProject-705690-1704903143\Test Cases\account_creation\invalid_name.tc</property>
  <property name="tag"></property>
  <property name="isOptional">false</property>
  <property name="currentRetryCount">0</property>
</record>
<record>
  <date>2024-02-06T07:48:10.489610200Z</date>
  <millis>1707205690489</millis>
  <nanos>610200</nanos>
  <sequence>7</sequence>
  <level>ERROR</level>
  <class>com.kms.katalon.core.logging.XmlKeywordLogger</class>
  <method>logMessage</method>
  <thread>1</thread>
  <message>Test Cases/account_creation/invalid_name has ERROR(s).
Reason:
java.io.FileNotFoundException: File &amp;apos;&amp;apos; does not exist</message>
  <nestedLevel>1</nestedLevel>
  <escapedJava>false</escapedJava>
  <property name="failed.exception.class">java.io.FileNotFoundException</property>
  <property name="failed.exception.stacktrace">java.io.FileNotFoundException: File &apos;&apos; does not exist
	at org.apache.commons.io.FileUtils.openInputStream(FileUtils.java:299)
	at org.apache.commons.io.FileUtils.readFileToString(FileUtils.java:1711)
	at org.apache.commons.io.FileUtils.readFileToString(FileUtils.java:1734)
	at com.kms.katalon.core.main.TestCaseMethodNodeCollector.&lt;init&gt;(TestCaseMethodNodeCollector.java:44)
	at com.kms.katalon.core.main.TestCaseExecutor.processScriptPreparationPhase(TestCaseExecutor.java:184)
	at com.kms.katalon.core.main.TestCaseExecutor.execute(TestCaseExecutor.java:259)
	at com.kms.katalon.core.common.CommonExecutor.accessTestCaseMainPhase(CommonExecutor.java:65)
	at com.kms.katalon.core.main.TestSuiteExecutor.accessTestSuiteMainPhase(TestSuiteExecutor.java:148)
	at com.kms.katalon.core.main.TestSuiteExecutor.execute(TestSuiteExecutor.java:106)
	at com.kms.katalon.core.main.TestCaseMain.startTestSuite(TestCaseMain.java:180)
	at TempTestSuite1707205675993.run(TempTestSuite1707205675993.groovy:36)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
</property>
  <property name="failed.exception.message">File &apos;&apos; does not exist</property>
</record>
<record>
  <date>2024-02-06T07:48:10.493609500Z</date>
  <millis>1707205690493</millis>
  <nanos>609500</nanos>
  <sequence>8</sequence>
  <level>END</level>
  <class>com.kms.katalon.core.logging.XmlKeywordLogger</class>
  <method>endTest</method>
  <thread>1</thread>
  <message>End Test Case : Test Cases/account_creation/invalid_name</message>
  <nestedLevel>0</nestedLevel>
  <escapedJava>false</escapedJava>
  <property name="executionBindingVariables">[]</property>
  <property name="applitools">false</property>
  <property name="remainingRetryCount">3</property>
  <property name="currentRetryCount">0</property>
  <property name="keywordsUsage">{
  &quot;web_ui&quot;: false,
  &quot;web_service&quot;: false,
  &quot;cucumber&quot;: false,
  &quot;builtin&quot;: false,
  &quot;mobile&quot;: false,
  &quot;windows&quot;: false,
  &quot;testng&quot;: false
}</property>
</record>
<record>
  <date>2024-02-06T07:48:10.499635700Z</date>
  <millis>1707205690499</millis>
  <nanos>635700</nanos>
  <sequence>9</sequence>
  <level>START</level>
  <class>com.kms.katalon.core.logging.XmlKeywordLogger</class>
  <method>startTest</method>
  <thread>1</thread>
  <message>Start Test Case : Test Cases/account_creation/invalid_name</message>
  <nestedLevel>1</nestedLevel>
  <escapedJava>false</escapedJava>
  <property name="remainingRetryCount">2</property>
  <property name="name">Test Cases/account_creation/invalid_name</property>
  <property name="description"></property>
  <property name="iteration"></property>
  <property name="id">Test Cases/account_creation/invalid_name</property>
  <property name="source">C:\Users\Devil\git\SI-GuidedProject-705690-1704903143\Test Cases\account_creation\invalid_name.tc</property>
  <property name="tag"></property>
  <property name="isOptional">false</property>
  <property name="currentRetryCount">1</property>
</record>
<record>
  <date>2024-02-06T07:48:10.510702600Z</date>
  <millis>1707205690510</millis>
  <nanos>702600</nanos>
  <sequence>11</sequence>
  <level>ERROR</level>
  <class>com.kms.katalon.core.logging.XmlKeywordLogger</class>
  <method>logMessage</method>
  <thread>1</thread>
  <message>Test Cases/account_creation/invalid_name has ERROR(s).
Reason:
java.io.FileNotFoundException: File &amp;apos;&amp;apos; does not exist</message>
  <nestedLevel>1</nestedLevel>
  <escapedJava>false</escapedJava>
  <property name="failed.exception.class">java.io.FileNotFoundException</property>
  <property name="failed.exception.stacktrace">java.io.FileNotFoundException: File &apos;&apos; does not exist
	at org.apache.commons.io.FileUtils.openInputStream(FileUtils.java:299)
	at org.apache.commons.io.FileUtils.readFileToString(FileUtils.java:1711)
	at org.apache.commons.io.FileUtils.readFileToString(FileUtils.java:1734)
	at com.kms.katalon.core.main.TestCaseMethodNodeCollector.&lt;init&gt;(TestCaseMethodNodeCollector.java:44)
	at com.kms.katalon.core.main.TestCaseExecutor.processScriptPreparationPhase(TestCaseExecutor.java:184)
	at com.kms.katalon.core.main.TestCaseExecutor.execute(TestCaseExecutor.java:259)
	at com.kms.katalon.core.main.TestCaseExecutor.execute(TestCaseExecutor.java:369)
	at com.kms.katalon.core.common.CommonExecutor.accessTestCaseMainPhase(CommonExecutor.java:65)
	at com.kms.katalon.core.main.TestSuiteExecutor.accessTestSuiteMainPhase(TestSuiteExecutor.java:148)
	at com.kms.katalon.core.main.TestSuiteExecutor.execute(TestSuiteExecutor.java:106)
	at com.kms.katalon.core.main.TestCaseMain.startTestSuite(TestCaseMain.java:180)
	at TempTestSuite1707205675993.run(TempTestSuite1707205675993.groovy:36)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
</property>
  <property name="failed.exception.message">File &apos;&apos; does not exist</property>
</record>
<record>
  <date>2024-02-06T07:48:10.513704700Z</date>
  <millis>1707205690513</millis>
  <nanos>704700</nanos>
  <sequence>12</sequence>
  <level>END</level>
  <class>com.kms.katalon.core.logging.XmlKeywordLogger</class>
  <method>endTest</method>
  <thread>1</thread>
  <message>End Test Case : Test Cases/account_creation/invalid_name</message>
  <nestedLevel>0</nestedLevel>
  <escapedJava>false</escapedJava>
  <property name="executionBindingVariables">[]</property>
  <property name="applitools">false</property>
  <property name="remainingRetryCount">2</property>
  <property name="currentRetryCount">1</property>
  <property name="keywordsUsage">{
  &quot;web_ui&quot;: false,
  &quot;web_service&quot;: false,
  &quot;cucumber&quot;: false,
  &quot;builtin&quot;: false,
  &quot;mobile&quot;: false,
  &quot;windows&quot;: false,
  &quot;testng&quot;: false
}</property>
</record>
<record>
  <date>2024-02-06T07:48:10.519877500Z</date>
  <millis>1707205690519</millis>
  <nanos>877500</nanos>
  <sequence>13</sequence>
  <level>START</level>
  <class>com.kms.katalon.core.logging.XmlKeywordLogger</class>
  <method>startTest</method>
  <thread>1</thread>
  <message>Start Test Case : Test Cases/account_creation/invalid_name</message>
  <nestedLevel>1</nestedLevel>
  <escapedJava>false</escapedJava>
  <property name="remainingRetryCount">1</property>
  <property name="name">Test Cases/account_creation/invalid_name</property>
  <property name="description"></property>
  <property name="iteration"></property>
  <property name="id">Test Cases/account_creation/invalid_name</property>
  <property name="source">C:\Users\Devil\git\SI-GuidedProject-705690-1704903143\Test Cases\account_creation\invalid_name.tc</property>
  <property name="tag"></property>
  <property name="isOptional">false</property>
  <property name="currentRetryCount">2</property>
</record>
<record>
  <date>2024-02-06T07:48:10.521875700Z</date>
  <millis>1707205690521</millis>
  <nanos>875700</nanos>
  <sequence>15</sequence>
  <level>ERROR</level>
  <class>com.kms.katalon.core.logging.XmlKeywordLogger</class>
  <method>logMessage</method>
  <thread>1</thread>
  <message>Test Cases/account_creation/invalid_name has ERROR(s).
Reason:
java.io.FileNotFoundException: File &amp;apos;&amp;apos; does not exist</message>
  <nestedLevel>1</nestedLevel>
  <escapedJava>false</escapedJava>
  <property name="failed.exception.class">java.io.FileNotFoundException</property>
  <property name="failed.exception.stacktrace">java.io.FileNotFoundException: File &apos;&apos; does not exist
	at org.apache.commons.io.FileUtils.openInputStream(FileUtils.java:299)
	at org.apache.commons.io.FileUtils.readFileToString(FileUtils.java:1711)
	at org.apache.commons.io.FileUtils.readFileToString(FileUtils.java:1734)
	at com.kms.katalon.core.main.TestCaseMethodNodeCollector.&lt;init&gt;(TestCaseMethodNodeCollector.java:44)
	at com.kms.katalon.core.main.TestCaseExecutor.processScriptPreparationPhase(TestCaseExecutor.java:184)
	at com.kms.katalon.core.main.TestCaseExecutor.execute(TestCaseExecutor.java:259)
	at com.kms.katalon.core.main.TestCaseExecutor.execute(TestCaseExecutor.java:369)
	at com.kms.katalon.core.main.TestCaseExecutor.execute(TestCaseExecutor.java:369)
	at com.kms.katalon.core.common.CommonExecutor.accessTestCaseMainPhase(CommonExecutor.java:65)
	at com.kms.katalon.core.main.TestSuiteExecutor.accessTestSuiteMainPhase(TestSuiteExecutor.java:148)
	at com.kms.katalon.core.main.TestSuiteExecutor.execute(TestSuiteExecutor.java:106)
	at com.kms.katalon.core.main.TestCaseMain.startTestSuite(TestCaseMain.java:180)
	at TempTestSuite1707205675993.run(TempTestSuite1707205675993.groovy:36)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
</property>
  <property name="failed.exception.message">File &apos;&apos; does not exist</property>
</record>
<record>
  <date>2024-02-06T07:48:10.524882800Z</date>
  <millis>1707205690524</millis>
  <nanos>882800</nanos>
  <sequence>16</sequence>
  <level>END</level>
  <class>com.kms.katalon.core.logging.XmlKeywordLogger</class>
  <method>endTest</method>
  <thread>1</thread>
  <message>End Test Case : Test Cases/account_creation/invalid_name</message>
  <nestedLevel>0</nestedLevel>
  <escapedJava>false</escapedJava>
  <property name="executionBindingVariables">[]</property>
  <property name="applitools">false</property>
  <property name="remainingRetryCount">1</property>
  <property name="currentRetryCount">2</property>
  <property name="keywordsUsage">{
  &quot;web_ui&quot;: false,
  &quot;web_service&quot;: false,
  &quot;cucumber&quot;: false,
  &quot;builtin&quot;: false,
  &quot;mobile&quot;: false,
  &quot;windows&quot;: false,
  &quot;testng&quot;: false
}</property>
</record>
<record>
  <date>2024-02-06T07:48:10.531877700Z</date>
  <millis>1707205690531</millis>
  <nanos>877700</nanos>
  <sequence>17</sequence>
  <level>START</level>
  <class>com.kms.katalon.core.logging.XmlKeywordLogger</class>
  <method>startTest</method>
  <thread>1</thread>
  <message>Start Test Case : Test Cases/account_creation/invalid_name</message>
  <nestedLevel>1</nestedLevel>
  <escapedJava>false</escapedJava>
  <property name="remainingRetryCount">0</property>
  <property name="name">Test Cases/account_creation/invalid_name</property>
  <property name="description"></property>
  <property name="iteration"></property>
  <property name="id">Test Cases/account_creation/invalid_name</property>
  <property name="source">C:\Users\Devil\git\SI-GuidedProject-705690-1704903143\Test Cases\account_creation\invalid_name.tc</property>
  <property name="tag"></property>
  <property name="isOptional">false</property>
  <property name="currentRetryCount">3</property>
</record>
<record>
  <date>2024-02-06T07:48:10.534876500Z</date>
  <millis>1707205690534</millis>
  <nanos>876500</nanos>
  <sequence>19</sequence>
  <level>ERROR</level>
  <class>com.kms.katalon.core.logging.XmlKeywordLogger</class>
  <method>logMessage</method>
  <thread>1</thread>
  <message>Test Cases/account_creation/invalid_name has ERROR(s).
Reason:
java.io.FileNotFoundException: File &amp;apos;&amp;apos; does not exist</message>
  <nestedLevel>1</nestedLevel>
  <escapedJava>false</escapedJava>
  <property name="failed.exception.class">java.io.FileNotFoundException</property>
  <property name="failed.exception.stacktrace">java.io.FileNotFoundException: File &apos;&apos; does not exist
	at org.apache.commons.io.FileUtils.openInputStream(FileUtils.java:299)
	at org.apache.commons.io.FileUtils.readFileToString(FileUtils.java:1711)
	at org.apache.commons.io.FileUtils.readFileToString(FileUtils.java:1734)
	at com.kms.katalon.core.main.TestCaseMethodNodeCollector.&lt;init&gt;(TestCaseMethodNodeCollector.java:44)
	at com.kms.katalon.core.main.TestCaseExecutor.processScriptPreparationPhase(TestCaseExecutor.java:184)
	at com.kms.katalon.core.main.TestCaseExecutor.execute(TestCaseExecutor.java:259)
	at com.kms.katalon.core.main.TestCaseExecutor.execute(TestCaseExecutor.java:369)
	at com.kms.katalon.core.main.TestCaseExecutor.execute(TestCaseExecutor.java:369)
	at com.kms.katalon.core.main.TestCaseExecutor.execute(TestCaseExecutor.java:369)
	at com.kms.katalon.core.common.CommonExecutor.accessTestCaseMainPhase(CommonExecutor.java:65)
	at com.kms.katalon.core.main.TestSuiteExecutor.accessTestSuiteMainPhase(TestSuiteExecutor.java:148)
	at com.kms.katalon.core.main.TestSuiteExecutor.execute(TestSuiteExecutor.java:106)
	at com.kms.katalon.core.main.TestCaseMain.startTestSuite(TestCaseMain.java:180)
	at TempTestSuite1707205675993.run(TempTestSuite1707205675993.groovy:36)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
</property>
  <property name="failed.exception.message">File &apos;&apos; does not exist</property>
</record>
<record>
  <date>2024-02-06T07:48:10.537657500Z</date>
  <millis>1707205690537</millis>
  <nanos>657500</nanos>
  <sequence>20</sequence>
  <level>END</level>
  <class>com.kms.katalon.core.logging.XmlKeywordLogger</class>
  <method>endTest</method>
  <thread>1</thread>
  <message>End Test Case : Test Cases/account_creation/invalid_name</message>
  <nestedLevel>0</nestedLevel>
  <escapedJava>false</escapedJava>
  <property name="executionBindingVariables">[]</property>
  <property name="applitools">false</property>
  <property name="remainingRetryCount">0</property>
  <property name="currentRetryCount">3</property>
  <property name="keywordsUsage">{
  &quot;web_ui&quot;: false,
  &quot;web_service&quot;: false,
  &quot;cucumber&quot;: false,
  &quot;builtin&quot;: false,
  &quot;mobile&quot;: false,
  &quot;windows&quot;: false,
  &quot;testng&quot;: false
}</property>
</record>
<record>
  <date>2024-02-06T07:48:10.555660400Z</date>
  <millis>1707205690555</millis>
  <nanos>660400</nanos>
  <sequence>21</sequence>
  <level>START</level>
  <class>com.kms.katalon.core.logging.XmlKeywordLogger</class>
  <method>startTest</method>
  <thread>1</thread>
  <message>Start Test Case : Test Cases/account_creation/invalid_phone_number</message>
  <nestedLevel>1</nestedLevel>
  <escapedJava>false</escapedJava>
  <property name="remainingRetryCount">3</property>
  <property name="name">Test Cases/account_creation/invalid_phone_number</property>
  <property name="description"></property>
  <property name="iteration"></property>
  <property name="id">Test Cases/account_creation/invalid_phone_number</property>
  <property name="source">C:\Users\Devil\git\SI-GuidedProject-705690-1704903143\Test Cases\account_creation\invalid_phone_number.tc</property>
  <property name="tag"></property>
  <property name="isOptional">false</property>
  <property name="currentRetryCount">0</property>
</record>
<record>
  <date>2024-02-06T07:48:10.557658400Z</date>
  <millis>1707205690557</millis>
  <nanos>658400</nanos>
  <sequence>23</sequence>
  <level>ERROR</level>
  <class>com.kms.katalon.core.logging.XmlKeywordLogger</class>
  <method>logMessage</method>
  <thread>1</thread>
  <message>Test Cases/account_creation/invalid_phone_number has ERROR(s).
Reason:
java.io.FileNotFoundException: File &amp;apos;&amp;apos; does not exist</message>
  <nestedLevel>1</nestedLevel>
  <escapedJava>false</escapedJava>
  <property name="failed.exception.class">java.io.FileNotFoundException</property>
  <property name="failed.exception.stacktrace">java.io.FileNotFoundException: File &apos;&apos; does not exist
	at org.apache.commons.io.FileUtils.openInputStream(FileUtils.java:299)
	at org.apache.commons.io.FileUtils.readFileToString(FileUtils.java:1711)
	at org.apache.commons.io.FileUtils.readFileToString(FileUtils.java:1734)
	at com.kms.katalon.core.main.TestCaseMethodNodeCollector.&lt;init&gt;(TestCaseMethodNodeCollector.java:44)
	at com.kms.katalon.core.main.TestCaseExecutor.processScriptPreparationPhase(TestCaseExecutor.java:184)
	at com.kms.katalon.core.main.TestCaseExecutor.execute(TestCaseExecutor.java:259)
	at com.kms.katalon.core.common.CommonExecutor.accessTestCaseMainPhase(CommonExecutor.java:65)
	at com.kms.katalon.core.main.TestSuiteExecutor.accessTestSuiteMainPhase(TestSuiteExecutor.java:148)
	at com.kms.katalon.core.main.TestSuiteExecutor.execute(TestSuiteExecutor.java:106)
	at com.kms.katalon.core.main.TestCaseMain.startTestSuite(TestCaseMain.java:180)
	at TempTestSuite1707205675993.run(TempTestSuite1707205675993.groovy:36)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
</property>
  <property name="failed.exception.message">File &apos;&apos; does not exist</property>
</record>
<record>
  <date>2024-02-06T07:48:10.559659400Z</date>
  <millis>1707205690559</millis>
  <nanos>659400</nanos>
  <sequence>24</sequence>
  <level>END</level>
  <class>com.kms.katalon.core.logging.XmlKeywordLogger</class>
  <method>endTest</method>
  <thread>1</thread>
  <message>End Test Case : Test Cases/account_creation/invalid_phone_number</message>
  <nestedLevel>0</nestedLevel>
  <escapedJava>false</escapedJava>
  <property name="executionBindingVariables">[]</property>
  <property name="applitools">false</property>
  <property name="remainingRetryCount">3</property>
  <property name="currentRetryCount">0</property>
  <property name="keywordsUsage">{
  &quot;web_ui&quot;: false,
  &quot;web_service&quot;: false,
  &quot;cucumber&quot;: false,
  &quot;builtin&quot;: false,
  &quot;mobile&quot;: false,
  &quot;windows&quot;: false,
  &quot;testng&quot;: false
}</property>
</record>
<record>
  <date>2024-02-06T07:48:10.563746600Z</date>
  <millis>1707205690563</millis>
  <nanos>746600</nanos>
  <sequence>25</sequence>
  <level>START</level>
  <class>com.kms.katalon.core.logging.XmlKeywordLogger</class>
  <method>startTest</method>
  <thread>1</thread>
  <message>Start Test Case : Test Cases/account_creation/invalid_phone_number</message>
  <nestedLevel>1</nestedLevel>
  <escapedJava>false</escapedJava>
  <property name="remainingRetryCount">2</property>
  <property name="name">Test Cases/account_creation/invalid_phone_number</property>
  <property name="description"></property>
  <property name="iteration"></property>
  <property name="id">Test Cases/account_creation/invalid_phone_number</property>
  <property name="source">C:\Users\Devil\git\SI-GuidedProject-705690-1704903143\Test Cases\account_creation\invalid_phone_number.tc</property>
  <property name="tag"></property>
  <property name="isOptional">false</property>
  <property name="currentRetryCount">1</property>
</record>
<record>
  <date>2024-02-06T07:48:10.563746600Z</date>
  <millis>1707205690563</millis>
  <nanos>746600</nanos>
  <sequence>27</sequence>
  <level>ERROR</level>
  <class>com.kms.katalon.core.logging.XmlKeywordLogger</class>
  <method>logMessage</method>
  <thread>1</thread>
  <message>Test Cases/account_creation/invalid_phone_number has ERROR(s).
Reason:
java.io.FileNotFoundException: File &amp;apos;&amp;apos; does not exist</message>
  <nestedLevel>1</nestedLevel>
  <escapedJava>false</escapedJava>
  <property name="failed.exception.class">java.io.FileNotFoundException</property>
  <property name="failed.exception.stacktrace">java.io.FileNotFoundException: File &apos;&apos; does not exist
	at org.apache.commons.io.FileUtils.openInputStream(FileUtils.java:299)
	at org.apache.commons.io.FileUtils.readFileToString(FileUtils.java:1711)
	at org.apache.commons.io.FileUtils.readFileToString(FileUtils.java:1734)
	at com.kms.katalon.core.main.TestCaseMethodNodeCollector.&lt;init&gt;(TestCaseMethodNodeCollector.java:44)
	at com.kms.katalon.core.main.TestCaseExecutor.processScriptPreparationPhase(TestCaseExecutor.java:184)
	at com.kms.katalon.core.main.TestCaseExecutor.execute(TestCaseExecutor.java:259)
	at com.kms.katalon.core.main.TestCaseExecutor.execute(TestCaseExecutor.java:369)
	at com.kms.katalon.core.common.CommonExecutor.accessTestCaseMainPhase(CommonExecutor.java:65)
	at com.kms.katalon.core.main.TestSuiteExecutor.accessTestSuiteMainPhase(TestSuiteExecutor.java:148)
	at com.kms.katalon.core.main.TestSuiteExecutor.execute(TestSuiteExecutor.java:106)
	at com.kms.katalon.core.main.TestCaseMain.startTestSuite(TestCaseMain.java:180)
	at TempTestSuite1707205675993.run(TempTestSuite1707205675993.groovy:36)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
</property>
  <property name="failed.exception.message">File &apos;&apos; does not exist</property>
</record>
<record>
  <date>2024-02-06T07:48:10.563746600Z</date>
  <millis>1707205690563</millis>
  <nanos>746600</nanos>
  <sequence>28</sequence>
  <level>END</level>
  <class>com.kms.katalon.core.logging.XmlKeywordLogger</class>
  <method>endTest</method>
  <thread>1</thread>
  <message>End Test Case : Test Cases/account_creation/invalid_phone_number</message>
  <nestedLevel>0</nestedLevel>
  <escapedJava>false</escapedJava>
  <property name="executionBindingVariables">[]</property>
  <property name="applitools">false</property>
  <property name="remainingRetryCount">2</property>
  <property name="currentRetryCount">1</property>
  <property name="keywordsUsage">{
  &quot;web_ui&quot;: false,
  &quot;web_service&quot;: false,
  &quot;cucumber&quot;: false,
  &quot;builtin&quot;: false,
  &quot;mobile&quot;: false,
  &quot;windows&quot;: false,
  &quot;testng&quot;: false
}</property>
</record>
<record>
  <date>2024-02-06T07:48:10.563746600Z</date>
  <millis>1707205690563</millis>
  <nanos>746600</nanos>
  <sequence>29</sequence>
  <level>START</level>
  <class>com.kms.katalon.core.logging.XmlKeywordLogger</class>
  <method>startTest</method>
  <thread>1</thread>
  <message>Start Test Case : Test Cases/account_creation/invalid_phone_number</message>
  <nestedLevel>1</nestedLevel>
  <escapedJava>false</escapedJava>
  <property name="remainingRetryCount">1</property>
  <property name="name">Test Cases/account_creation/invalid_phone_number</property>
  <property name="description"></property>
  <property name="iteration"></property>
  <property name="id">Test Cases/account_creation/invalid_phone_number</property>
  <property name="source">C:\Users\Devil\git\SI-GuidedProject-705690-1704903143\Test Cases\account_creation\invalid_phone_number.tc</property>
  <property name="tag"></property>
  <property name="isOptional">false</property>
  <property name="currentRetryCount">2</property>
</record>
<record>
  <date>2024-02-06T07:48:10.563746600Z</date>
  <millis>1707205690563</millis>
  <nanos>746600</nanos>
  <sequence>31</sequence>
  <level>ERROR</level>
  <class>com.kms.katalon.core.logging.XmlKeywordLogger</class>
  <method>logMessage</method>
  <thread>1</thread>
  <message>Test Cases/account_creation/invalid_phone_number has ERROR(s).
Reason:
java.io.FileNotFoundException: File &amp;apos;&amp;apos; does not exist</message>
  <nestedLevel>1</nestedLevel>
  <escapedJava>false</escapedJava>
  <property name="failed.exception.class">java.io.FileNotFoundException</property>
  <property name="failed.exception.stacktrace">java.io.FileNotFoundException: File &apos;&apos; does not exist
	at org.apache.commons.io.FileUtils.openInputStream(FileUtils.java:299)
	at org.apache.commons.io.FileUtils.readFileToString(FileUtils.java:1711)
	at org.apache.commons.io.FileUtils.readFileToString(FileUtils.java:1734)
	at com.kms.katalon.core.main.TestCaseMethodNodeCollector.&lt;init&gt;(TestCaseMethodNodeCollector.java:44)
	at com.kms.katalon.core.main.TestCaseExecutor.processScriptPreparationPhase(TestCaseExecutor.java:184)
	at com.kms.katalon.core.main.TestCaseExecutor.execute(TestCaseExecutor.java:259)
	at com.kms.katalon.core.main.TestCaseExecutor.execute(TestCaseExecutor.java:369)
	at com.kms.katalon.core.main.TestCaseExecutor.execute(TestCaseExecutor.java:369)
	at com.kms.katalon.core.common.CommonExecutor.accessTestCaseMainPhase(CommonExecutor.java:65)
	at com.kms.katalon.core.main.TestSuiteExecutor.accessTestSuiteMainPhase(TestSuiteExecutor.java:148)
	at com.kms.katalon.core.main.TestSuiteExecutor.execute(TestSuiteExecutor.java:106)
	at com.kms.katalon.core.main.TestCaseMain.startTestSuite(TestCaseMain.java:180)
	at TempTestSuite1707205675993.run(TempTestSuite1707205675993.groovy:36)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
</property>
  <property name="failed.exception.message">File &apos;&apos; does not exist</property>
</record>
<record>
  <date>2024-02-06T07:48:10.579374900Z</date>
  <millis>1707205690579</millis>
  <nanos>374900</nanos>
  <sequence>32</sequence>
  <level>END</level>
  <class>com.kms.katalon.core.logging.XmlKeywordLogger</class>
  <method>endTest</method>
  <thread>1</thread>
  <message>End Test Case : Test Cases/account_creation/invalid_phone_number</message>
  <nestedLevel>0</nestedLevel>
  <escapedJava>false</escapedJava>
  <property name="executionBindingVariables">[]</property>
  <property name="applitools">false</property>
  <property name="remainingRetryCount">1</property>
  <property name="currentRetryCount">2</property>
  <property name="keywordsUsage">{
  &quot;web_ui&quot;: false,
  &quot;web_service&quot;: false,
  &quot;cucumber&quot;: false,
  &quot;builtin&quot;: false,
  &quot;mobile&quot;: false,
  &quot;windows&quot;: false,
  &quot;testng&quot;: false
}</property>
</record>
<record>
  <date>2024-02-06T07:48:10.579374900Z</date>
  <millis>1707205690579</millis>
  <nanos>374900</nanos>
  <sequence>33</sequence>
  <level>START</level>
  <class>com.kms.katalon.core.logging.XmlKeywordLogger</class>
  <method>startTest</method>
  <thread>1</thread>
  <message>Start Test Case : Test Cases/account_creation/invalid_phone_number</message>
  <nestedLevel>1</nestedLevel>
  <escapedJava>false</escapedJava>
  <property name="remainingRetryCount">0</property>
  <property name="name">Test Cases/account_creation/invalid_phone_number</property>
  <property name="description"></property>
  <property name="iteration"></property>
  <property name="id">Test Cases/account_creation/invalid_phone_number</property>
  <property name="source">C:\Users\Devil\git\SI-GuidedProject-705690-1704903143\Test Cases\account_creation\invalid_phone_number.tc</property>
  <property name="tag"></property>
  <property name="isOptional">false</property>
  <property name="currentRetryCount">3</property>
</record>
<record>
  <date>2024-02-06T07:48:10.579374900Z</date>
  <millis>1707205690579</millis>
  <nanos>374900</nanos>
  <sequence>35</sequence>
  <level>ERROR</level>
  <class>com.kms.katalon.core.logging.XmlKeywordLogger</class>
  <method>logMessage</method>
  <thread>1</thread>
  <message>Test Cases/account_creation/invalid_phone_number has ERROR(s).
Reason:
java.io.FileNotFoundException: File &amp;apos;&amp;apos; does not exist</message>
  <nestedLevel>1</nestedLevel>
  <escapedJava>false</escapedJava>
  <property name="failed.exception.class">java.io.FileNotFoundException</property>
  <property name="failed.exception.stacktrace">java.io.FileNotFoundException: File &apos;&apos; does not exist
	at org.apache.commons.io.FileUtils.openInputStream(FileUtils.java:299)
	at org.apache.commons.io.FileUtils.readFileToString(FileUtils.java:1711)
	at org.apache.commons.io.FileUtils.readFileToString(FileUtils.java:1734)
	at com.kms.katalon.core.main.TestCaseMethodNodeCollector.&lt;init&gt;(TestCaseMethodNodeCollector.java:44)
	at com.kms.katalon.core.main.TestCaseExecutor.processScriptPreparationPhase(TestCaseExecutor.java:184)
	at com.kms.katalon.core.main.TestCaseExecutor.execute(TestCaseExecutor.java:259)
	at com.kms.katalon.core.main.TestCaseExecutor.execute(TestCaseExecutor.java:369)
	at com.kms.katalon.core.main.TestCaseExecutor.execute(TestCaseExecutor.java:369)
	at com.kms.katalon.core.main.TestCaseExecutor.execute(TestCaseExecutor.java:369)
	at com.kms.katalon.core.common.CommonExecutor.accessTestCaseMainPhase(CommonExecutor.java:65)
	at com.kms.katalon.core.main.TestSuiteExecutor.accessTestSuiteMainPhase(TestSuiteExecutor.java:148)
	at com.kms.katalon.core.main.TestSuiteExecutor.execute(TestSuiteExecutor.java:106)
	at com.kms.katalon.core.main.TestCaseMain.startTestSuite(TestCaseMain.java:180)
	at TempTestSuite1707205675993.run(TempTestSuite1707205675993.groovy:36)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
</property>
  <property name="failed.exception.message">File &apos;&apos; does not exist</property>
</record>
<record>
  <date>2024-02-06T07:48:10.579374900Z</date>
  <millis>1707205690579</millis>
  <nanos>374900</nanos>
  <sequence>36</sequence>
  <level>END</level>
  <class>com.kms.katalon.core.logging.XmlKeywordLogger</class>
  <method>endTest</method>
  <thread>1</thread>
  <message>End Test Case : Test Cases/account_creation/invalid_phone_number</message>
  <nestedLevel>0</nestedLevel>
  <escapedJava>false</escapedJava>
  <property name="executionBindingVariables">[]</property>
  <property name="applitools">false</property>
  <property name="remainingRetryCount">0</property>
  <property name="currentRetryCount">3</property>
  <property name="keywordsUsage">{
  &quot;web_ui&quot;: false,
  &quot;web_service&quot;: false,
  &quot;cucumber&quot;: false,
  &quot;builtin&quot;: false,
  &quot;mobile&quot;: false,
  &quot;windows&quot;: false,
  &quot;testng&quot;: false
}</property>
</record>
<record>
  <date>2024-02-06T07:48:10.605649900Z</date>
  <millis>1707205690605</millis>
  <nanos>649900</nanos>
  <sequence>37</sequence>
  <level>START</level>
  <class>com.kms.katalon.core.logging.XmlKeywordLogger</class>
  <method>startTest</method>
  <thread>1</thread>
  <message>Start Test Case : Test Cases/account_creation/no_password</message>
  <nestedLevel>1</nestedLevel>
  <escapedJava>false</escapedJava>
  <property name="remainingRetryCount">3</property>
  <property name="name">Test Cases/account_creation/no_password</property>
  <property name="description"></property>
  <property name="iteration"></property>
  <property name="id">Test Cases/account_creation/no_password</property>
  <property name="source">C:\Users\Devil\git\SI-GuidedProject-705690-1704903143\Test Cases\account_creation\no_password.tc</property>
  <property name="tag"></property>
  <property name="isOptional">false</property>
  <property name="currentRetryCount">0</property>
</record>
<record>
  <date>2024-02-06T07:48:10.605649900Z</date>
  <millis>1707205690605</millis>
  <nanos>649900</nanos>
  <sequence>39</sequence>
  <level>ERROR</level>
  <class>com.kms.katalon.core.logging.XmlKeywordLogger</class>
  <method>logMessage</method>
  <thread>1</thread>
  <message>Test Cases/account_creation/no_password has ERROR(s).
Reason:
java.io.FileNotFoundException: File &amp;apos;&amp;apos; does not exist</message>
  <nestedLevel>1</nestedLevel>
  <escapedJava>false</escapedJava>
  <property name="failed.exception.class">java.io.FileNotFoundException</property>
  <property name="failed.exception.stacktrace">java.io.FileNotFoundException: File &apos;&apos; does not exist
	at org.apache.commons.io.FileUtils.openInputStream(FileUtils.java:299)
	at org.apache.commons.io.FileUtils.readFileToString(FileUtils.java:1711)
	at org.apache.commons.io.FileUtils.readFileToString(FileUtils.java:1734)
	at com.kms.katalon.core.main.TestCaseMethodNodeCollector.&lt;init&gt;(TestCaseMethodNodeCollector.java:44)
	at com.kms.katalon.core.main.TestCaseExecutor.processScriptPreparationPhase(TestCaseExecutor.java:184)
	at com.kms.katalon.core.main.TestCaseExecutor.execute(TestCaseExecutor.java:259)
	at com.kms.katalon.core.common.CommonExecutor.accessTestCaseMainPhase(CommonExecutor.java:65)
	at com.kms.katalon.core.main.TestSuiteExecutor.accessTestSuiteMainPhase(TestSuiteExecutor.java:148)
	at com.kms.katalon.core.main.TestSuiteExecutor.execute(TestSuiteExecutor.java:106)
	at com.kms.katalon.core.main.TestCaseMain.startTestSuite(TestCaseMain.java:180)
	at TempTestSuite1707205675993.run(TempTestSuite1707205675993.groovy:36)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
</property>
  <property name="failed.exception.message">File &apos;&apos; does not exist</property>
</record>
<record>
  <date>2024-02-06T07:48:10.605649900Z</date>
  <millis>1707205690605</millis>
  <nanos>649900</nanos>
  <sequence>40</sequence>
  <level>END</level>
  <class>com.kms.katalon.core.logging.XmlKeywordLogger</class>
  <method>endTest</method>
  <thread>1</thread>
  <message>End Test Case : Test Cases/account_creation/no_password</message>
  <nestedLevel>0</nestedLevel>
  <escapedJava>false</escapedJava>
  <property name="executionBindingVariables">[]</property>
  <property name="applitools">false</property>
  <property name="remainingRetryCount">3</property>
  <property name="currentRetryCount">0</property>
  <property name="keywordsUsage">{
  &quot;web_ui&quot;: false,
  &quot;web_service&quot;: false,
  &quot;cucumber&quot;: false,
  &quot;builtin&quot;: false,
  &quot;mobile&quot;: false,
  &quot;windows&quot;: false,
  &quot;testng&quot;: false
}</property>
</record>
<record>
  <date>2024-02-06T07:48:10.620320700Z</date>
  <millis>1707205690620</millis>
  <nanos>320700</nanos>
  <sequence>41</sequence>
  <level>START</level>
  <class>com.kms.katalon.core.logging.XmlKeywordLogger</class>
  <method>startTest</method>
  <thread>1</thread>
  <message>Start Test Case : Test Cases/account_creation/no_password</message>
  <nestedLevel>1</nestedLevel>
  <escapedJava>false</escapedJava>
  <property name="remainingRetryCount">2</property>
  <property name="name">Test Cases/account_creation/no_password</property>
  <property name="description"></property>
  <property name="iteration"></property>
  <property name="id">Test Cases/account_creation/no_password</property>
  <property name="source">C:\Users\Devil\git\SI-GuidedProject-705690-1704903143\Test Cases\account_creation\no_password.tc</property>
  <property name="tag"></property>
  <property name="isOptional">false</property>
  <property name="currentRetryCount">1</property>
</record>
<record>
  <date>2024-02-06T07:48:10.621984800Z</date>
  <millis>1707205690621</millis>
  <nanos>984800</nanos>
  <sequence>43</sequence>
  <level>ERROR</level>
  <class>com.kms.katalon.core.logging.XmlKeywordLogger</class>
  <method>logMessage</method>
  <thread>1</thread>
  <message>Test Cases/account_creation/no_password has ERROR(s).
Reason:
java.io.FileNotFoundException: File &amp;apos;&amp;apos; does not exist</message>
  <nestedLevel>1</nestedLevel>
  <escapedJava>false</escapedJava>
  <property name="failed.exception.class">java.io.FileNotFoundException</property>
  <property name="failed.exception.stacktrace">java.io.FileNotFoundException: File &apos;&apos; does not exist
	at org.apache.commons.io.FileUtils.openInputStream(FileUtils.java:299)
	at org.apache.commons.io.FileUtils.readFileToString(FileUtils.java:1711)
	at org.apache.commons.io.FileUtils.readFileToString(FileUtils.java:1734)
	at com.kms.katalon.core.main.TestCaseMethodNodeCollector.&lt;init&gt;(TestCaseMethodNodeCollector.java:44)
	at com.kms.katalon.core.main.TestCaseExecutor.processScriptPreparationPhase(TestCaseExecutor.java:184)
	at com.kms.katalon.core.main.TestCaseExecutor.execute(TestCaseExecutor.java:259)
	at com.kms.katalon.core.main.TestCaseExecutor.execute(TestCaseExecutor.java:369)
	at com.kms.katalon.core.common.CommonExecutor.accessTestCaseMainPhase(CommonExecutor.java:65)
	at com.kms.katalon.core.main.TestSuiteExecutor.accessTestSuiteMainPhase(TestSuiteExecutor.java:148)
	at com.kms.katalon.core.main.TestSuiteExecutor.execute(TestSuiteExecutor.java:106)
	at com.kms.katalon.core.main.TestCaseMain.startTestSuite(TestCaseMain.java:180)
	at TempTestSuite1707205675993.run(TempTestSuite1707205675993.groovy:36)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
</property>
  <property name="failed.exception.message">File &apos;&apos; does not exist</property>
</record>
<record>
  <date>2024-02-06T07:48:10.621984800Z</date>
  <millis>1707205690621</millis>
  <nanos>984800</nanos>
  <sequence>44</sequence>
  <level>END</level>
  <class>com.kms.katalon.core.logging.XmlKeywordLogger</class>
  <method>endTest</method>
  <thread>1</thread>
  <message>End Test Case : Test Cases/account_creation/no_password</message>
  <nestedLevel>0</nestedLevel>
  <escapedJava>false</escapedJava>
  <property name="executionBindingVariables">[]</property>
  <property name="applitools">false</property>
  <property name="remainingRetryCount">2</property>
  <property name="currentRetryCount">1</property>
  <property name="keywordsUsage">{
  &quot;web_ui&quot;: false,
  &quot;web_service&quot;: false,
  &quot;cucumber&quot;: false,
  &quot;builtin&quot;: false,
  &quot;mobile&quot;: false,
  &quot;windows&quot;: false,
  &quot;testng&quot;: false
}</property>
</record>
<record>
  <date>2024-02-06T07:48:10.633003500Z</date>
  <millis>1707205690633</millis>
  <nanos>3500</nanos>
  <sequence>45</sequence>
  <level>START</level>
  <class>com.kms.katalon.core.logging.XmlKeywordLogger</class>
  <method>startTest</method>
  <thread>1</thread>
  <message>Start Test Case : Test Cases/account_creation/no_password</message>
  <nestedLevel>1</nestedLevel>
  <escapedJava>false</escapedJava>
  <property name="remainingRetryCount">1</property>
  <property name="name">Test Cases/account_creation/no_password</property>
  <property name="description"></property>
  <property name="iteration"></property>
  <property name="id">Test Cases/account_creation/no_password</property>
  <property name="source">C:\Users\Devil\git\SI-GuidedProject-705690-1704903143\Test Cases\account_creation\no_password.tc</property>
  <property name="tag"></property>
  <property name="isOptional">false</property>
  <property name="currentRetryCount">2</property>
</record>
<record>
  <date>2024-02-06T07:48:10.636003800Z</date>
  <millis>1707205690636</millis>
  <nanos>3800</nanos>
  <sequence>47</sequence>
  <level>ERROR</level>
  <class>com.kms.katalon.core.logging.XmlKeywordLogger</class>
  <method>logMessage</method>
  <thread>1</thread>
  <message>Test Cases/account_creation/no_password has ERROR(s).
Reason:
java.io.FileNotFoundException: File &amp;apos;&amp;apos; does not exist</message>
  <nestedLevel>1</nestedLevel>
  <escapedJava>false</escapedJava>
  <property name="failed.exception.class">java.io.FileNotFoundException</property>
  <property name="failed.exception.stacktrace">java.io.FileNotFoundException: File &apos;&apos; does not exist
	at org.apache.commons.io.FileUtils.openInputStream(FileUtils.java:299)
	at org.apache.commons.io.FileUtils.readFileToString(FileUtils.java:1711)
	at org.apache.commons.io.FileUtils.readFileToString(FileUtils.java:1734)
	at com.kms.katalon.core.main.TestCaseMethodNodeCollector.&lt;init&gt;(TestCaseMethodNodeCollector.java:44)
	at com.kms.katalon.core.main.TestCaseExecutor.processScriptPreparationPhase(TestCaseExecutor.java:184)
	at com.kms.katalon.core.main.TestCaseExecutor.execute(TestCaseExecutor.java:259)
	at com.kms.katalon.core.main.TestCaseExecutor.execute(TestCaseExecutor.java:369)
	at com.kms.katalon.core.main.TestCaseExecutor.execute(TestCaseExecutor.java:369)
	at com.kms.katalon.core.common.CommonExecutor.accessTestCaseMainPhase(CommonExecutor.java:65)
	at com.kms.katalon.core.main.TestSuiteExecutor.accessTestSuiteMainPhase(TestSuiteExecutor.java:148)
	at com.kms.katalon.core.main.TestSuiteExecutor.execute(TestSuiteExecutor.java:106)
	at com.kms.katalon.core.main.TestCaseMain.startTestSuite(TestCaseMain.java:180)
	at TempTestSuite1707205675993.run(TempTestSuite1707205675993.groovy:36)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
</property>
  <property name="failed.exception.message">File &apos;&apos; does not exist</property>
</record>
<record>
  <date>2024-02-06T07:48:10.638004700Z</date>
  <millis>1707205690638</millis>
  <nanos>4700</nanos>
  <sequence>48</sequence>
  <level>END</level>
  <class>com.kms.katalon.core.logging.XmlKeywordLogger</class>
  <method>endTest</method>
  <thread>1</thread>
  <message>End Test Case : Test Cases/account_creation/no_password</message>
  <nestedLevel>0</nestedLevel>
  <escapedJava>false</escapedJava>
  <property name="executionBindingVariables">[]</property>
  <property name="applitools">false</property>
  <property name="remainingRetryCount">1</property>
  <property name="currentRetryCount">2</property>
  <property name="keywordsUsage">{
  &quot;web_ui&quot;: false,
  &quot;web_service&quot;: false,
  &quot;cucumber&quot;: false,
  &quot;builtin&quot;: false,
  &quot;mobile&quot;: false,
  &quot;windows&quot;: false,
  &quot;testng&quot;: false
}</property>
</record>
<record>
  <date>2024-02-06T07:48:10.643005500Z</date>
  <millis>1707205690643</millis>
  <nanos>5500</nanos>
  <sequence>49</sequence>
  <level>START</level>
  <class>com.kms.katalon.core.logging.XmlKeywordLogger</class>
  <method>startTest</method>
  <thread>1</thread>
  <message>Start Test Case : Test Cases/account_creation/no_password</message>
  <nestedLevel>1</nestedLevel>
  <escapedJava>false</escapedJava>
  <property name="remainingRetryCount">0</property>
  <property name="name">Test Cases/account_creation/no_password</property>
  <property name="description"></property>
  <property name="iteration"></property>
  <property name="id">Test Cases/account_creation/no_password</property>
  <property name="source">C:\Users\Devil\git\SI-GuidedProject-705690-1704903143\Test Cases\account_creation\no_password.tc</property>
  <property name="tag"></property>
  <property name="isOptional">false</property>
  <property name="currentRetryCount">3</property>
</record>
<record>
  <date>2024-02-06T07:48:10.646005500Z</date>
  <millis>1707205690646</millis>
  <nanos>5500</nanos>
  <sequence>51</sequence>
  <level>ERROR</level>
  <class>com.kms.katalon.core.logging.XmlKeywordLogger</class>
  <method>logMessage</method>
  <thread>1</thread>
  <message>Test Cases/account_creation/no_password has ERROR(s).
Reason:
java.io.FileNotFoundException: File &amp;apos;&amp;apos; does not exist</message>
  <nestedLevel>1</nestedLevel>
  <escapedJava>false</escapedJava>
  <property name="failed.exception.class">java.io.FileNotFoundException</property>
  <property name="failed.exception.stacktrace">java.io.FileNotFoundException: File &apos;&apos; does not exist
	at org.apache.commons.io.FileUtils.openInputStream(FileUtils.java:299)
	at org.apache.commons.io.FileUtils.readFileToString(FileUtils.java:1711)
	at org.apache.commons.io.FileUtils.readFileToString(FileUtils.java:1734)
	at com.kms.katalon.core.main.TestCaseMethodNodeCollector.&lt;init&gt;(TestCaseMethodNodeCollector.java:44)
	at com.kms.katalon.core.main.TestCaseExecutor.processScriptPreparationPhase(TestCaseExecutor.java:184)
	at com.kms.katalon.core.main.TestCaseExecutor.execute(TestCaseExecutor.java:259)
	at com.kms.katalon.core.main.TestCaseExecutor.execute(TestCaseExecutor.java:369)
	at com.kms.katalon.core.main.TestCaseExecutor.execute(TestCaseExecutor.java:369)
	at com.kms.katalon.core.main.TestCaseExecutor.execute(TestCaseExecutor.java:369)
	at com.kms.katalon.core.common.CommonExecutor.accessTestCaseMainPhase(CommonExecutor.java:65)
	at com.kms.katalon.core.main.TestSuiteExecutor.accessTestSuiteMainPhase(TestSuiteExecutor.java:148)
	at com.kms.katalon.core.main.TestSuiteExecutor.execute(TestSuiteExecutor.java:106)
	at com.kms.katalon.core.main.TestCaseMain.startTestSuite(TestCaseMain.java:180)
	at TempTestSuite1707205675993.run(TempTestSuite1707205675993.groovy:36)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
</property>
  <property name="failed.exception.message">File &apos;&apos; does not exist</property>
</record>
<record>
  <date>2024-02-06T07:48:10.648007800Z</date>
  <millis>1707205690648</millis>
  <nanos>7800</nanos>
  <sequence>52</sequence>
  <level>END</level>
  <class>com.kms.katalon.core.logging.XmlKeywordLogger</class>
  <method>endTest</method>
  <thread>1</thread>
  <message>End Test Case : Test Cases/account_creation/no_password</message>
  <nestedLevel>0</nestedLevel>
  <escapedJava>false</escapedJava>
  <property name="executionBindingVariables">[]</property>
  <property name="applitools">false</property>
  <property name="remainingRetryCount">0</property>
  <property name="currentRetryCount">3</property>
  <property name="keywordsUsage">{
  &quot;web_ui&quot;: false,
  &quot;web_service&quot;: false,
  &quot;cucumber&quot;: false,
  &quot;builtin&quot;: false,
  &quot;mobile&quot;: false,
  &quot;windows&quot;: false,
  &quot;testng&quot;: false
}</property>
</record>
<record>
  <date>2024-02-06T07:48:10.665009200Z</date>
  <millis>1707205690665</millis>
  <nanos>9200</nanos>
  <sequence>53</sequence>
  <level>START</level>
  <class>com.kms.katalon.core.logging.XmlKeywordLogger</class>
  <method>startTest</method>
  <thread>1</thread>
  <message>Start Test Case : Test Cases/account_creation/phone_number_exists001</message>
  <nestedLevel>1</nestedLevel>
  <escapedJava>false</escapedJava>
  <property name="remainingRetryCount">3</property>
  <property name="name">Test Cases/account_creation/phone_number_exists001</property>
  <property name="description"></property>
  <property name="iteration"></property>
  <property name="id">Test Cases/account_creation/phone_number_exists001</property>
  <property name="source">C:\Users\Devil\git\SI-GuidedProject-705690-1704903143\Test Cases\account_creation\phone_number_exists001.tc</property>
  <property name="tag"></property>
  <property name="isOptional">false</property>
  <property name="currentRetryCount">0</property>
</record>
<record>
  <date>2024-02-06T07:48:10.665781600Z</date>
  <millis>1707205690665</millis>
  <nanos>781600</nanos>
  <sequence>55</sequence>
  <level>ERROR</level>
  <class>com.kms.katalon.core.logging.XmlKeywordLogger</class>
  <method>logMessage</method>
  <thread>1</thread>
  <message>Test Cases/account_creation/phone_number_exists001 has ERROR(s).
Reason:
java.io.FileNotFoundException: File &amp;apos;&amp;apos; does not exist</message>
  <nestedLevel>1</nestedLevel>
  <escapedJava>false</escapedJava>
  <property name="failed.exception.class">java.io.FileNotFoundException</property>
  <property name="failed.exception.stacktrace">java.io.FileNotFoundException: File &apos;&apos; does not exist
	at org.apache.commons.io.FileUtils.openInputStream(FileUtils.java:299)
	at org.apache.commons.io.FileUtils.readFileToString(FileUtils.java:1711)
	at org.apache.commons.io.FileUtils.readFileToString(FileUtils.java:1734)
	at com.kms.katalon.core.main.TestCaseMethodNodeCollector.&lt;init&gt;(TestCaseMethodNodeCollector.java:44)
	at com.kms.katalon.core.main.TestCaseExecutor.processScriptPreparationPhase(TestCaseExecutor.java:184)
	at com.kms.katalon.core.main.TestCaseExecutor.execute(TestCaseExecutor.java:259)
	at com.kms.katalon.core.common.CommonExecutor.accessTestCaseMainPhase(CommonExecutor.java:65)
	at com.kms.katalon.core.main.TestSuiteExecutor.accessTestSuiteMainPhase(TestSuiteExecutor.java:148)
	at com.kms.katalon.core.main.TestSuiteExecutor.execute(TestSuiteExecutor.java:106)
	at com.kms.katalon.core.main.TestCaseMain.startTestSuite(TestCaseMain.java:180)
	at TempTestSuite1707205675993.run(TempTestSuite1707205675993.groovy:36)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
</property>
  <property name="failed.exception.message">File &apos;&apos; does not exist</property>
</record>
<record>
  <date>2024-02-06T07:48:10.665781600Z</date>
  <millis>1707205690665</millis>
  <nanos>781600</nanos>
  <sequence>56</sequence>
  <level>END</level>
  <class>com.kms.katalon.core.logging.XmlKeywordLogger</class>
  <method>endTest</method>
  <thread>1</thread>
  <message>End Test Case : Test Cases/account_creation/phone_number_exists001</message>
  <nestedLevel>0</nestedLevel>
  <escapedJava>false</escapedJava>
  <property name="executionBindingVariables">[]</property>
  <property name="applitools">false</property>
  <property name="remainingRetryCount">3</property>
  <property name="currentRetryCount">0</property>
  <property name="keywordsUsage">{
  &quot;web_ui&quot;: false,
  &quot;web_service&quot;: false,
  &quot;cucumber&quot;: false,
  &quot;builtin&quot;: false,
  &quot;mobile&quot;: false,
  &quot;windows&quot;: false,
  &quot;testng&quot;: false
}</property>
</record>
<record>
  <date>2024-02-06T07:48:10.665781600Z</date>
  <millis>1707205690665</millis>
  <nanos>781600</nanos>
  <sequence>57</sequence>
  <level>START</level>
  <class>com.kms.katalon.core.logging.XmlKeywordLogger</class>
  <method>startTest</method>
  <thread>1</thread>
  <message>Start Test Case : Test Cases/account_creation/phone_number_exists001</message>
  <nestedLevel>1</nestedLevel>
  <escapedJava>false</escapedJava>
  <property name="remainingRetryCount">2</property>
  <property name="name">Test Cases/account_creation/phone_number_exists001</property>
  <property name="description"></property>
  <property name="iteration"></property>
  <property name="id">Test Cases/account_creation/phone_number_exists001</property>
  <property name="source">C:\Users\Devil\git\SI-GuidedProject-705690-1704903143\Test Cases\account_creation\phone_number_exists001.tc</property>
  <property name="tag"></property>
  <property name="isOptional">false</property>
  <property name="currentRetryCount">1</property>
</record>
<record>
  <date>2024-02-06T07:48:10.665781600Z</date>
  <millis>1707205690665</millis>
  <nanos>781600</nanos>
  <sequence>59</sequence>
  <level>ERROR</level>
  <class>com.kms.katalon.core.logging.XmlKeywordLogger</class>
  <method>logMessage</method>
  <thread>1</thread>
  <message>Test Cases/account_creation/phone_number_exists001 has ERROR(s).
Reason:
java.io.FileNotFoundException: File &amp;apos;&amp;apos; does not exist</message>
  <nestedLevel>1</nestedLevel>
  <escapedJava>false</escapedJava>
  <property name="failed.exception.class">java.io.FileNotFoundException</property>
  <property name="failed.exception.stacktrace">java.io.FileNotFoundException: File &apos;&apos; does not exist
	at org.apache.commons.io.FileUtils.openInputStream(FileUtils.java:299)
	at org.apache.commons.io.FileUtils.readFileToString(FileUtils.java:1711)
	at org.apache.commons.io.FileUtils.readFileToString(FileUtils.java:1734)
	at com.kms.katalon.core.main.TestCaseMethodNodeCollector.&lt;init&gt;(TestCaseMethodNodeCollector.java:44)
	at com.kms.katalon.core.main.TestCaseExecutor.processScriptPreparationPhase(TestCaseExecutor.java:184)
	at com.kms.katalon.core.main.TestCaseExecutor.execute(TestCaseExecutor.java:259)
	at com.kms.katalon.core.main.TestCaseExecutor.execute(TestCaseExecutor.java:369)
	at com.kms.katalon.core.common.CommonExecutor.accessTestCaseMainPhase(CommonExecutor.java:65)
	at com.kms.katalon.core.main.TestSuiteExecutor.accessTestSuiteMainPhase(TestSuiteExecutor.java:148)
	at com.kms.katalon.core.main.TestSuiteExecutor.execute(TestSuiteExecutor.java:106)
	at com.kms.katalon.core.main.TestCaseMain.startTestSuite(TestCaseMain.java:180)
	at TempTestSuite1707205675993.run(TempTestSuite1707205675993.groovy:36)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
</property>
  <property name="failed.exception.message">File &apos;&apos; does not exist</property>
</record>
<record>
  <date>2024-02-06T07:48:10.665781600Z</date>
  <millis>1707205690665</millis>
  <nanos>781600</nanos>
  <sequence>60</sequence>
  <level>END</level>
  <class>com.kms.katalon.core.logging.XmlKeywordLogger</class>
  <method>endTest</method>
  <thread>1</thread>
  <message>End Test Case : Test Cases/account_creation/phone_number_exists001</message>
  <nestedLevel>0</nestedLevel>
  <escapedJava>false</escapedJava>
  <property name="executionBindingVariables">[]</property>
  <property name="applitools">false</property>
  <property name="remainingRetryCount">2</property>
  <property name="currentRetryCount">1</property>
  <property name="keywordsUsage">{
  &quot;web_ui&quot;: false,
  &quot;web_service&quot;: false,
  &quot;cucumber&quot;: false,
  &quot;builtin&quot;: false,
  &quot;mobile&quot;: false,
  &quot;windows&quot;: false,
  &quot;testng&quot;: false
}</property>
</record>
<record>
  <date>2024-02-06T07:48:10.682073200Z</date>
  <millis>1707205690682</millis>
  <nanos>73200</nanos>
  <sequence>61</sequence>
  <level>START</level>
  <class>com.kms.katalon.core.logging.XmlKeywordLogger</class>
  <method>startTest</method>
  <thread>1</thread>
  <message>Start Test Case : Test Cases/account_creation/phone_number_exists001</message>
  <nestedLevel>1</nestedLevel>
  <escapedJava>false</escapedJava>
  <property name="remainingRetryCount">1</property>
  <property name="name">Test Cases/account_creation/phone_number_exists001</property>
  <property name="description"></property>
  <property name="iteration"></property>
  <property name="id">Test Cases/account_creation/phone_number_exists001</property>
  <property name="source">C:\Users\Devil\git\SI-GuidedProject-705690-1704903143\Test Cases\account_creation\phone_number_exists001.tc</property>
  <property name="tag"></property>
  <property name="isOptional">false</property>
  <property name="currentRetryCount">2</property>
</record>
<record>
  <date>2024-02-06T07:48:10.682073200Z</date>
  <millis>1707205690682</millis>
  <nanos>73200</nanos>
  <sequence>63</sequence>
  <level>ERROR</level>
  <class>com.kms.katalon.core.logging.XmlKeywordLogger</class>
  <method>logMessage</method>
  <thread>1</thread>
  <message>Test Cases/account_creation/phone_number_exists001 has ERROR(s).
Reason:
java.io.FileNotFoundException: File &amp;apos;&amp;apos; does not exist</message>
  <nestedLevel>1</nestedLevel>
  <escapedJava>false</escapedJava>
  <property name="failed.exception.class">java.io.FileNotFoundException</property>
  <property name="failed.exception.stacktrace">java.io.FileNotFoundException: File &apos;&apos; does not exist
	at org.apache.commons.io.FileUtils.openInputStream(FileUtils.java:299)
	at org.apache.commons.io.FileUtils.readFileToString(FileUtils.java:1711)
	at org.apache.commons.io.FileUtils.readFileToString(FileUtils.java:1734)
	at com.kms.katalon.core.main.TestCaseMethodNodeCollector.&lt;init&gt;(TestCaseMethodNodeCollector.java:44)
	at com.kms.katalon.core.main.TestCaseExecutor.processScriptPreparationPhase(TestCaseExecutor.java:184)
	at com.kms.katalon.core.main.TestCaseExecutor.execute(TestCaseExecutor.java:259)
	at com.kms.katalon.core.main.TestCaseExecutor.execute(TestCaseExecutor.java:369)
	at com.kms.katalon.core.main.TestCaseExecutor.execute(TestCaseExecutor.java:369)
	at com.kms.katalon.core.common.CommonExecutor.accessTestCaseMainPhase(CommonExecutor.java:65)
	at com.kms.katalon.core.main.TestSuiteExecutor.accessTestSuiteMainPhase(TestSuiteExecutor.java:148)
	at com.kms.katalon.core.main.TestSuiteExecutor.execute(TestSuiteExecutor.java:106)
	at com.kms.katalon.core.main.TestCaseMain.startTestSuite(TestCaseMain.java:180)
	at TempTestSuite1707205675993.run(TempTestSuite1707205675993.groovy:36)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
</property>
  <property name="failed.exception.message">File &apos;&apos; does not exist</property>
</record>
<record>
  <date>2024-02-06T07:48:10.682073200Z</date>
  <millis>1707205690682</millis>
  <nanos>73200</nanos>
  <sequence>64</sequence>
  <level>END</level>
  <class>com.kms.katalon.core.logging.XmlKeywordLogger</class>
  <method>endTest</method>
  <thread>1</thread>
  <message>End Test Case : Test Cases/account_creation/phone_number_exists001</message>
  <nestedLevel>0</nestedLevel>
  <escapedJava>false</escapedJava>
  <property name="executionBindingVariables">[]</property>
  <property name="applitools">false</property>
  <property name="remainingRetryCount">1</property>
  <property name="currentRetryCount">2</property>
  <property name="keywordsUsage">{
  &quot;web_ui&quot;: false,
  &quot;web_service&quot;: false,
  &quot;cucumber&quot;: false,
  &quot;builtin&quot;: false,
  &quot;mobile&quot;: false,
  &quot;windows&quot;: false,
  &quot;testng&quot;: false
}</property>
</record>
<record>
  <date>2024-02-06T07:48:10.682073200Z</date>
  <millis>1707205690682</millis>
  <nanos>73200</nanos>
  <sequence>65</sequence>
  <level>START</level>
  <class>com.kms.katalon.core.logging.XmlKeywordLogger</class>
  <method>startTest</method>
  <thread>1</thread>
  <message>Start Test Case : Test Cases/account_creation/phone_number_exists001</message>
  <nestedLevel>1</nestedLevel>
  <escapedJava>false</escapedJava>
  <property name="remainingRetryCount">0</property>
  <property name="name">Test Cases/account_creation/phone_number_exists001</property>
  <property name="description"></property>
  <property name="iteration"></property>
  <property name="id">Test Cases/account_creation/phone_number_exists001</property>
  <property name="source">C:\Users\Devil\git\SI-GuidedProject-705690-1704903143\Test Cases\account_creation\phone_number_exists001.tc</property>
  <property name="tag"></property>
  <property name="isOptional">false</property>
  <property name="currentRetryCount">3</property>
</record>
<record>
  <date>2024-02-06T07:48:10.697699100Z</date>
  <millis>1707205690697</millis>
  <nanos>699100</nanos>
  <sequence>67</sequence>
  <level>ERROR</level>
  <class>com.kms.katalon.core.logging.XmlKeywordLogger</class>
  <method>logMessage</method>
  <thread>1</thread>
  <message>Test Cases/account_creation/phone_number_exists001 has ERROR(s).
Reason:
java.io.FileNotFoundException: File &amp;apos;&amp;apos; does not exist</message>
  <nestedLevel>1</nestedLevel>
  <escapedJava>false</escapedJava>
  <property name="failed.exception.class">java.io.FileNotFoundException</property>
  <property name="failed.exception.stacktrace">java.io.FileNotFoundException: File &apos;&apos; does not exist
	at org.apache.commons.io.FileUtils.openInputStream(FileUtils.java:299)
	at org.apache.commons.io.FileUtils.readFileToString(FileUtils.java:1711)
	at org.apache.commons.io.FileUtils.readFileToString(FileUtils.java:1734)
	at com.kms.katalon.core.main.TestCaseMethodNodeCollector.&lt;init&gt;(TestCaseMethodNodeCollector.java:44)
	at com.kms.katalon.core.main.TestCaseExecutor.processScriptPreparationPhase(TestCaseExecutor.java:184)
	at com.kms.katalon.core.main.TestCaseExecutor.execute(TestCaseExecutor.java:259)
	at com.kms.katalon.core.main.TestCaseExecutor.execute(TestCaseExecutor.java:369)
	at com.kms.katalon.core.main.TestCaseExecutor.execute(TestCaseExecutor.java:369)
	at com.kms.katalon.core.main.TestCaseExecutor.execute(TestCaseExecutor.java:369)
	at com.kms.katalon.core.common.CommonExecutor.accessTestCaseMainPhase(CommonExecutor.java:65)
	at com.kms.katalon.core.main.TestSuiteExecutor.accessTestSuiteMainPhase(TestSuiteExecutor.java:148)
	at com.kms.katalon.core.main.TestSuiteExecutor.execute(TestSuiteExecutor.java:106)
	at com.kms.katalon.core.main.TestCaseMain.startTestSuite(TestCaseMain.java:180)
	at TempTestSuite1707205675993.run(TempTestSuite1707205675993.groovy:36)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
</property>
  <property name="failed.exception.message">File &apos;&apos; does not exist</property>
</record>
<record>
  <date>2024-02-06T07:48:10.697699100Z</date>
  <millis>1707205690697</millis>
  <nanos>699100</nanos>
  <sequence>68</sequence>
  <level>END</level>
  <class>com.kms.katalon.core.logging.XmlKeywordLogger</class>
  <method>endTest</method>
  <thread>1</thread>
  <message>End Test Case : Test Cases/account_creation/phone_number_exists001</message>
  <nestedLevel>0</nestedLevel>
  <escapedJava>false</escapedJava>
  <property name="executionBindingVariables">[]</property>
  <property name="applitools">false</property>
  <property name="remainingRetryCount">0</property>
  <property name="currentRetryCount">3</property>
  <property name="keywordsUsage">{
  &quot;web_ui&quot;: false,
  &quot;web_service&quot;: false,
  &quot;cucumber&quot;: false,
  &quot;builtin&quot;: false,
  &quot;mobile&quot;: false,
  &quot;windows&quot;: false,
  &quot;testng&quot;: false
}</property>
</record>
<record>
  <date>2024-02-06T07:48:10.697699100Z</date>
  <millis>1707205690697</millis>
  <nanos>699100</nanos>
  <sequence>69</sequence>
  <level>START</level>
  <class>com.kms.katalon.core.logging.XmlKeywordLogger</class>
  <method>startKeyword</method>
  <thread>1</thread>
  <message>Start listener action : exportKatalonReports</message>
  <nestedLevel>0</nestedLevel>
  <escapedJava>false</escapedJava>
</record>
<record>
  <date>2024-02-06T07:48:11.259634100Z</date>
  <millis>1707205691259</millis>
  <nanos>634100</nanos>
  <sequence>179</sequence>
  <level>INFO</level>
  <class>com.kms.katalon.core.logging.XmlKeywordLogger</class>
  <method>logMessage</method>
  <thread>1</thread>
  <message>Start generating HTML report folder at: C:\Users\Devil\git\SI-GuidedProject-705690-1704903143\Reports\20240206_131755\account_creation_errors\account_creation_errors\20240206_131755...</message>
  <nestedLevel>0</nestedLevel>
  <escapedJava>false</escapedJava>
</record>
<record>
  <date>2024-02-06T07:48:11.290889800Z</date>
  <millis>1707205691290</millis>
  <nanos>889800</nanos>
  <sequence>180</sequence>
  <level>INFO</level>
  <class>com.kms.katalon.core.logging.XmlKeywordLogger</class>
  <method>logMessage</method>
  <thread>1</thread>
  <message>HTML report generated</message>
  <nestedLevel>0</nestedLevel>
  <escapedJava>false</escapedJava>
</record>
<record>
  <date>2024-02-06T07:48:11.290889800Z</date>
  <millis>1707205691290</millis>
  <nanos>889800</nanos>
  <sequence>181</sequence>
  <level>INFO</level>
  <class>com.kms.katalon.core.logging.XmlKeywordLogger</class>
  <method>logMessage</method>
  <thread>1</thread>
  <message>Start generating CSV report folder at: C:\Users\Devil\git\SI-GuidedProject-705690-1704903143\Reports\20240206_131755\account_creation_errors\account_creation_errors\20240206_131755...</message>
  <nestedLevel>0</nestedLevel>
  <escapedJava>false</escapedJava>
</record>
<record>
  <date>2024-02-06T07:48:11.333532500Z</date>
  <millis>1707205691333</millis>
  <nanos>532500</nanos>
  <sequence>182</sequence>
  <level>INFO</level>
  <class>com.kms.katalon.core.logging.XmlKeywordLogger</class>
  <method>logMessage</method>
  <thread>1</thread>
  <message>CSV report generated</message>
  <nestedLevel>0</nestedLevel>
  <escapedJava>false</escapedJava>
</record>
<record>
  <date>2024-02-06T07:48:12.090350Z</date>
  <millis>1707205692090</millis>
  <nanos>350000</nanos>
  <sequence>184</sequence>
  <level>END</level>
  <class>com.kms.katalon.core.logging.XmlKeywordLogger</class>
  <method>endKeyword</method>
  <thread>1</thread>
  <message>End listener action : exportKatalonReports</message>
  <nestedLevel>0</nestedLevel>
  <escapedJava>false</escapedJava>
</record>
<record>
  <date>2024-02-06T07:48:12.180055800Z</date>
  <millis>1707205692180</millis>
  <nanos>55800</nanos>
  <sequence>185</sequence>
  <level>END</level>
  <class>com.kms.katalon.core.logging.XmlKeywordLogger</class>
  <method>endSuite</method>
  <thread>1</thread>
  <message>End Test Suite : Test Suites/account_creation_errors/account_creation_errors</message>
  <nestedLevel>0</nestedLevel>
  <escapedJava>false</escapedJava>
</record>
</log>
